block-producer/src/config.rs

```diff
@@ -0,0 +1,54 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+
+pub const HOST: &str = "localhost";
+// defined as: sum(ord(c)**p for (p, c) in enumerate('miden-block-producer', 1)) % 2**16
+pub const PORT: u16 = 48046;
+pub const ENV_PREFIX: &str = "MIDEN_BLOCK_PRODUCER";
+pub const CONFIG_FILENAME: &str = "miden-block-producer.toml";
+
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+pub struct Endpoint {
+    /// The host device the server will bind to.
+    pub host: String,
+    /// The port number to bind the server to.
+    pub port: u16,
+}
```

Comment:
I think this is the 3rd place where we define `Endpoint` struct. I wonder if it would make sense to define it once in some central location. No need to do this in this PR - but let's create an issue for this.

Reply:
https://github.com/0xPolygonMiden/miden-node/issues/76